### PR TITLE
Ui4

### PR DESCRIPTION
**Pull request by heemo521**
ui fix on logins, profile page, headers, lobby, waiting room, create room

### CODE REVIEW

File: src/features/WaitingRoom.js [hunk 1]
```diff
@@ -32,89 +30,131 @@ useEffect(() => {
...
+              sx={{
+                p: '0.5rem',
+                width: '50%',
+                ml: '1rem',
+                fontSize: { sm: '1rem', md: '1.wrem' },
+                outline: '1px solid white',
+              }}
+              onClick={() => setShow(true)}
+            >
+              Start Game
```
Comment (crypto-bender) [inline src/features/WaitingRoom.js:140]: are we adding this back in?

**Review by crypto-bender — APPROVED** [review]
Looks good, @kesang20 says we can remove the extra buttons in subsequent PR's.

**Review by kesang20 — APPROVED** [review]
good to go Looks great!

File: src/features/WaitingRoom.js (hunk 1 above)
Comment (heemo521) [reply src/features/WaitingRoom.js:140]: No. I added it unintentionally. Kesang is going to revert the change when he merges his part.

**Pull request merged**